deployer/utils.py

```diff
@@ -48,29 +48,35 @@ def decrypt_file(encrypted_path):
         ])
         yield f.name
 
-def flatten_list(inputlist):
+def add_staff(staff, user_list):
     """
-    Flatten a list of strings and lists into a flat list of strings.
-    This will only work 1-level deep.
+    Add the staff ids to the existing user list
     """
-    if isinstance(inputlist, str):
-        inputlist = [inputlist]
-    out = []
-    for ii in inputlist:
-        if isinstance(ii, str):
-            out.append(ii)
-        else:
-            out.extend(ii)
-    return out
+    if isinstance(user_list, str):
+        user_list = [user_list]
+
+    staff_list = []
+
+    if "staff_google_ids" in user_list:
+        user_list = user_list.remove("staff_google_ids")
+        staff_list =  staff["googleIds"]
+    elif "staff_github_ids" in user_list:
+        user_list = user_list.remove("staff_github_ids")
+        staff_list =  staff["githubIds"]
+
```

Comment:
```suggestion
    for staff_list_type, staff_ids in staff.items():
        if staff_list_type in user_list:
            user_list = user_list.remove(staff_list_type)
            staff_list = staff_ids
            break
```

Reply:
What do you think about re-using the keys that are in our `staff.yaml` file, so that if we add more there it will automatically be checked here?